Kitodo/src/main/webapp/pages/metadataEditor.xhtml

```diff
@@ -33,6 +33,7 @@
         <f:viewAction action="#{DataEditorForm.initMetadataEditor()}"/>
         <f:viewAction
                 action="#{DataEditorForm.structurePanel.setActiveTabs(DataEditorForm.structurePanel.separateMedia ? '0,1' : '0')}"/>
+        <f:importConstants type="org.kitodo.production.forms.dataeditor.StructurePanel" />
```

Comment:
If I remember correctly, you placed this tag here instead of `logicalStructure.xhtml` because it needs to be contained in a `<f:metadata>` tag, which can only be added to a template client, but not a template, right? (see https://jakarta.ee/specifications/faces/2.3/vdldoc/f/metadata.html for details) 

Apparently, this can be avoided by using the OmniFaces variant of `importConstants` instead of the vanilla JSF variant: https://showcase.omnifaces.org/taghandlers/importConstants

So using 
`<o:importConstants type="org.kitodo.production.forms.dataeditor.StructurePanel" />`
in `logicalStructure.xhtml`
instead of 
`<f:importConstants type="org.kitodo.production.forms.dataeditor.StructurePanel" />`
in `metadataEditor.xhtml` should work.

Reply:
This `<importConstants>` will be also needed in  the `physicalStructure.xhtml` file. So I think it's better to make this import line ( `<f:importConstants type="org.kitodo.production.forms.dataeditor.StructurePanel"/>`) in `metadataEditor.xhtml`  for both files `physicalStructure.xhtml` and `logicalStructure.xhtml`.